health-services/project-factory/src/server/utils/sheetManageUtils.ts

```
@@ -0,0 +1,595 @@
+}
+
+
+const markAsExpired = (resources: any[], currentTime: number, userUuid: string) =>
+    resources.map((resource) => {
+        const audit = resource.auditDetails || {};
+        return {
+            ...resource,
+            status: generatedResourceStatuses.expired,
+            count: null,
+            auditDetails: {
+                createdTime: parseInt(audit.createdTime ?? `${currentTime}`),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)